source/_posts/2019-09-27-warehouse-3_hexo-util-1-3-1.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Official plugins warehouse 3.0.0 & hexo-util 1.3.1 released
```

Comment:
`hexo-util 1.2.0, 1.3.0, 1.3.1 released` is better.

Reply:
Done